### PR TITLE
Users spaces

### PR DESCRIPTION
**Pull request by davidbgk**

### CODE REVIEW

File: repondeur/zam_repondeur/templates/space_detail.html [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+{% endblock %}
+
+{% block extra_nav %}
+    <li {% if owner == request.user %}class="selected"{% endif %}>
+        <a href="{{ request.resource_url(context.parent, request.user.email) }}">
+            Ma table
+        </a>
+    </li>
+    <li>
+        <a href="{{ request.resource_url(context.lecture_resource, 'amendements') }}">
```
Comment (davidbgk) [inline repondeur/zam_repondeur/templates/space_detail.html]: Maybe we should use `radar_url` here too?

File: repondeur/zam_repondeur/templates/macros.html [hunk 2]
```diff
@@ -45,12 +45,16 @@ <h3>
...
-            {% if event.details_template %}
-                {{ event.render_details() }}
+        <div class="what">
+            {% if event.details_template and event.details_template.template %}
+                <details {% if is_first  %}open{% endif %}>
+                    <summary>{{ event.render_summary() }}</summary>
+                    {{ event.render_details() }}
+                </details>
+            {% else %}
+                {{ event.render_summary() }}
```
Comment (davidbgk) [inline repondeur/zam_repondeur/templates/macros.html]: Maybe put that in a `<p>` tag?

File: repondeur/setup.cfg [hunk 3]
```diff
@@ -1,5 +1,6 @@
 [flake8]
 max_line_length = 88
+ignore = E302,W503
```
Comment (ronnix) [inline repondeur/setup.cfg]: 👍

File: repondeur/zam_repondeur/static/js/amendements.js [hunk 4]
```diff
@@ -307,13 +307,51 @@ function takeControlOverNativeJump() {
...
+
 function selectMultiple(checkboxes) {
-    const options = document.querySelector('.groupActions')
-    const checkboxesArray = Array.from(checkboxes)
-    checkboxesArray.forEach(checkbox => {
-        checkbox.addEventListener('click', e => {
-            const checkeds = checkboxesArray.filter((box) => box.checked)
-            options.style.display = (checkeds.length < 1) ? 'none' : 'flex'
+    const target = document.querySelector('.groupActions')
+    // Useful in case of (soft) refresh with already checked boxes.
```
Comment (ronnix) [inline repondeur/zam_repondeur/static/js/amendements.js]: What's a soft refresh exactly?

Comment (davidbgk) [reply repondeur/zam_repondeur/static/js/amendements.js]: Soft as opposed to hard where you hit `cmd+maj+r` which cleans up previously checked checkboxes

File: repondeur/zam_repondeur/views/download.py [hunk 5]
```diff
@@ -55,6 +62,47 @@ def download_amendements(context: LectureResource, request: Request) -> Response
...
+            f"{lecture.chambre}-{lecture.session}-{lecture.num_texte}-"
+            f"{lecture.organe}.pdf"
+        )
+        response.content_type = "application/pdf"
+        response.headers["Content-Disposition"] = f"attachment; filename={attach_name}"
+        return response
+
+
 @view_config(context=AmendementResource, name="download_amendement")
 def download_amendement(context: AmendementResource, request: Request) -> Response:
```
Comment (davidbgk) [inline repondeur/zam_repondeur/views/download.py]: @ronnix did you keep that view intentionally?

Comment (ronnix) [reply repondeur/zam_repondeur/views/download.py]: I almost removed it, but I wondered if we would need it, e.g. for printing a single one from my table ?

Happy to remove it if there's no use case...

Comment (ronnix) [reply repondeur/zam_repondeur/views/download.py]: I think I'll remove it. If we need it again, we could even reuse the other view, as printing a single amendement is just a special case of printing multiple ones.

**Pull request merged**